config/application.rb

```diff
@@ -90,5 +90,17 @@ class Application < Rails::Application
     }
 
     config.action_mailer.deliver_later_queue_name = :mailers
+
+    config.x.sections.editable = %i[
```

Comment:
Should these be configs? `editable_window` should probably have `days` in its name.

Reply:
I think they should be configs. Anything against? Happy to add `_days` suffix to editable_window

Reply:
Maybe even `working_days`

Reply:
@inulty-dfe Thoughts on this? To me it doesn't look like it "configures" something. You are using it in a class method in `Section` and it's self contained there. Not even used in the initializer. I don't really mind where it sits, just found it a bit weird.

Reply:
Done